.circleci/config.yml

```diff
@@ -52,6 +52,18 @@ jobs:
       - run:
           name: Test server
           command: cd server && yarn && yarn test
+  deploy-server:
+    working_directory: ~/batch-10-tripen/server
+    docker:
+      - image: circleci/node:10.16.3
+    steps:
+      - checkout
+      - restore_cache:
+          key: v1-deps-server-cache-{{ checksum "./server/yarn.lock" }}
+      - deploy:
+          name: Deploy to heroku server
+          command:
+            git subtree push --prefix server https://heroku:$HEROKU_API_KEY@git.heroku.com/$HEROKU_APP_NAME.git master
```

Comment:
Have you guys created only a single environment? Is anything on master is in production env? Is there any staging environment?

Reply:
Have you guys created only a single environment? 
yes.
Is anything on master is in production env?
yes.
Is there any staging environment?
no.